jcl/src/com.ibm.management/share/classes/com/ibm/lang/management/RuntimeMXBean.java

```diff
@@ -68,4 +68,10 @@
 	 */
 	public double getVMGeneratedCPULoad();
 
+	/**
+	 * Returns current JVM state.
+	 *
+	 * @return VM State value - Active / Idle
+	 */
+	public String getVMState();
```

Comment:
I think this should have a more descriptive name, perhaps getVMIdleState().

I suggest adding some public constants to hold the "Active" and "Idle" strings, which can be used to programmatically test the state. Alternatively another API could be added - isVMIdle().

Reply:
Actually, having an API which returns English words doesn't seem appropriate. Better to return a constant from an enum.